README.md

```diff
@@ -1,94 +1,71 @@
-# ![Async Logo](https://raw.githubusercontent.com/asyncjs/assets/master/branding/logo/async-logo-32x32.png) AsyncJS Assets
+![Async Logo](./branding/logo/logo.png)
 
-**A collection of AsyncJS assets**
+This repository contains Async brand assets and guidelines. Feel free to use and customize these assets for your articles, talks, websites, and anything else you can think of.
 
-## Branding
+<p align="center">
+  <a href="#logo">Logo</a>&nbsp;&nbsp;&nbsp;
+  <a href="#colours">Colours</a>&nbsp;&nbsp;&nbsp;
+  <a href="#fonts">Fonts</a>&nbsp;&nbsp;&nbsp;
+</p>
```

Comment:
I think these would be easier to access if they were directly under the description and structured like:

```md
- [Logo](#logo)
- [Colours](#colours)
- [Fonts](#fonts)
```